<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="camera_img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="live_img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>528</width>
      <height>396</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="prog_img">
    <property name="geometry">
     <rect>
      <x>538</x>
      <y>490</y>
      <width>420</width>
      <height>210</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="log_text">
    <property name="geometry">
     <rect>
      <x>538</x>
      <y>721</y>
      <width>420</width>
      <height>210</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
   </widget>
   <widget class="QFrame" name="track_control_frame">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="play_pause_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="skip_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip</string>
     </property>
    </widget>
    <widget class="QSlider" name="vol_slider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="vol_slider_text">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>50%</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="EMMA_control_frame">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pause_EMMA_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause EMMA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="register_user_btn">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>90</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Register new user</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
